ext/drreg/drreg.h

```diff
@@ -317,6 +317,13 @@ typedef enum {
      * scratch registers are then restored prior to each application instruction.
      */
     DRREG_IGNORE_CONTROL_FLOW = 0x002,
+    /**
+     * Turns off register restoration at the end of the block.
+     * Note that it is still required that registers have their original
+     * values at the end of a basic block. Therefore, restoration needs
+     * to be handled by the user manually, usually via drreg_restore_all().
+     */
+    DRREG_IGNORE_BB_END_RESTORE = 0x004,
```

Comment:
How about something stronger than "ignore": maybe `DRREG_USER_RESTORES_AT_BB_END` or sthg similar to capture that drreg is not doing it but the user has to?

Reply:
Done